lib/Models/BaseMaps/BaseMapsModel.ts

```diff
@@ -104,7 +115,8 @@ export class BaseMapsModel extends CreateModel(BaseMapsTraits) {
       });
     }
 
-    updateModelFromJson(this, stratumId, rest).pushErrorTo(errors);
+    if (isJsonObject(rest))
+      updateModelFromJson(this, stratumId, rest).pushErrorTo(errors);
```

Comment:
To me these lines seem either too silent (if `rest` isn't a JsonObject, it's not an error) or too defensive (can `rest` be something other than a `JsonObject` here if `newBaseMaps` is a `BaseMapsJson`?

Reply:
Added an error message.

`rest` should be `JsonObject` - but I was getting type issues

<img width="1162" alt="image" src="https://user-images.githubusercontent.com/6187649/162865993-bcf42f21-a36e-4cd6-bb30-fef2de693bb9.png">